src/OSPSuite.UI/Controls/UxRichEditControl.cs

```diff
@@ -15,5 +18,37 @@ private void onUnhandledException(object sender, RichEditUnhandledExceptionEvent
       {
          richEditUnhandledExceptionEventArgs.Handled = true;
       }
+
```

Comment:
The RichEdit did not behave exactly as the label did during resize so IXtraResizableControl had to be implemented.

Do you think we should create a separate control for this purpose - making a richedit act like a label?